### PR TITLE
Update regions for Austria, Germany, Liechtenstein, Switzerland

### PR DESCRIPTION
**Pull request by fan711**
- updated regions for Austria, Germany, Switzerland to the official local name
- added regions for Liechtenstein

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Edit a contact and find new and updated regions in the 'State' dropdown

### CODE REVIEW

**Review by npracht — CHANGES_REQUESTED** [review]
Hi @fan711 !
Thanks for your first contribution.
Few feedbacks:
1. I think we need to add migration for existing saved values in DB. Example here: https://github.com/mautic/mautic/pull/9204/files
2. This is a BC for people using API for example. The old value (for edited values) will be rejected. @escopecz is that correct ? If yes we have to include this in a minor / major ?
3. Otherwise it seems correctly relying on https://en.wikipedia.org/wiki/ISO_3166-2 👍🏽

**Comment by fan711** [comment]
Thank you @npracht for reviewing. Added a migration. I agree, it will affect api users and that was my actual motivation to update these regions. I'll be happy to post a new pr against 5.x since rebasing seems too cumbersome.

**Comment by npracht** [comment]
Let's wait for @escopecz feedback about which branch to target. And for the rest i must say good job ;)

**Review by npracht — APPROVED** [review]

File: app/migrations/Version20230525202700.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+final class Version20230525202700 extends AbstractMauticMigration
+{
+    /**
+     * @throws SkipMigration
+     */
+    public function preUp(Schema $schema): void
```
Comment (escopecz) [inline app/migrations/Version20230525202700.php:16]: Feel free to remove this method completely. It should run in all instances at least once. No need to check if it should run.

**Comment by escopecz** [comment]
@npracht good point. Let's close this PR as the 4.x branch is now dead. There won't be any new Mautic 4 minor releases. Only patches.

Please raise this against the 5.x branch and let's merge it before the 5.0.0 is released. Also, please document the changes in https://github.com/mautic/mautic/blob/5.x/UPGRADE-5.0.md

**Pull request closed without merging**